git/remote.py

```diff
@@ -391,7 +391,7 @@ def __init__(self, repo, name):
 
     def __getattr__(self, attr):
         """Allows to call this instance like
-        remote.special( \*args, \*\*kwargs) to call git-remote special self.name"""
+        remote.special( \\*args, \\*\\*kwargs) to call git-remote special self.name"""
```

Comment:
I don't have any say in this repo so I don't mind either way. Just a preference and for homogeneity.